hardware/Comply-to-the-Regulations.md

```diff
@@ -0,0 +1,394 @@
+   </div>
+   
+ - RX Spurious Emissions
+   In the Rx and other operation modes, The power of any unwanted emission in spurious domain should not exceed the values given in following table:
+   <div align="center">
+   <img src="files\HW-Comply-to-the-Regulations\Spurous-Values.png">  
+   </div>
+   
+## 4.4. CE requirements in 2.4GHz band
+ETSI [EN300328](https://ec.europa.eu/growth/single-market/european-standards/harmonised-standards/red_en) mainly defines the 2.4GHz ISM band and using wide band modulation regulation for CE certification. It applies to wide band data transmission equipment in 2400MHz to 2483.5MHz band.
+
+ - Tx power limits
+   The RF output power is defined as the mean equivalent isotropically radiated power(EIRP) of the equipment during a transmission burst. 
+   The maximum allowed output power is +20dBm.
+
+ - Occupied Channel Bandwidth
+   It is defined as bandwidth containing 99% energy of the output power.
+   For FHSS systems:
+   - For equipment with EIRP ≤ +10 dBm, OCBW is simply declared by the supplier (Section 4.3.1.7.2)
+   - For equipment with EIRP > +10 dBm, OCBW is declared by the supplier, but it should be less than 5 MHz (Section 4.3.1.7.2)
+   For wideband digital modulation systems:
+   - For equipment with EIRP > +10 dBm, OCBW declared by the supplier, but less than 20 MHz (Section 4.3.2.6.2)
+
+ - TX Power Spectral Density
+   - Applies only to wideband digital modulation systems, and not FHSS systems
+   - PSD(max) ≤ +10 dBm / MHz (Section 4.3.2.2.2)
+
+ - TX Unwanted Emissions in Out-of-Band Domain
+   - OOB Domain defined for “near-by” signals (relative to edges of frequency band)
+   - Within 2*BW of 2.400-2.4835 GHz band edges, where BW = Occupied Channel Bandwidth
+   - At greater frequency offsets, the Spurious Domain limits apply
+   - OOB Domain Limits:
+     - -10 dBm/MHz EIRP, within 1*BW offset from band edge (Section 4.3.1.8.2)
+     - -20 dBm/MHz EIRP, within 1*BW to 2*BW offset from band edge (Section 4.3.1.8.2)
+
+ - TX Unwanted Emissions in Spurious Domain
+   - Spurious Domain defined for “far-away” signals (relative to edges of frequency band)
+   - More than 2*BW away from band edges,Where “BW” is the Occupied Channel Bandwidth (but minimum value of 1 MHz)
+   - At lesser frequency offsets, the Out-of-Band Domain limits apply
+   - Spurious Domain Limits: (Section 4.3.1.9.2): 1-12.75 GHz, -30 dBm EIRP, RBW=1MHz
+
+ - Rx sensitivity level
+   No general requirements. But it is defined in Zigbee and BLE protocols respectively.
+
+ - Blocking
+   - Desired signal at -30 dBm, Interferer at -30 dBm (CW) (Section 4.3.1.11.2)
+     - Relative blocking level much easier than BTLE spec
+     - Absolute blocking level slightly harder than BTLE spec
+   - But not applicable for systems with TX EIRP < +10dBm. I.e., not applicable to BTLE
+
+ - RX Spurious Emissions
+   - 30-1000 MHz, -57 dBm ERP, BW=100kHz (Section 4.3.1.10.2)
```

Comment:
ERP in this line, and EIRP in the next line.  Is that intended, or should these align?

Reply:
It is intended.